cdi2/pom.xml

```diff
@@ -0,0 +1,53 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <properties>
+        <openwebbeans.version>2.0.10</openwebbeans.version>
+    </properties>
+
+    <parent>
+        <groupId>io.cucumber</groupId>
+        <artifactId>cucumber-jvm</artifactId>
+        <version>4.3.2-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>cucumber-cdi2</artifactId>
+    <packaging>jar</packaging>
+    <name>Cucumber-JVM: CDI 2</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.cucumber</groupId>
+            <artifactId>cucumber-java</artifactId>
```

Comment:
I've just merged https://github.com/cucumber/cucumber-jvm/pull/1615. By implementing `io.cucumber.core.backend.ObjectFactory` only a dependency on `cucumber-core` is needed.